test/lib/global_uid_test.rb

```diff
@@ -618,6 +618,14 @@ def parent_child_fork_values
       assert_equal uids.uniq, uids
     end
 
+    it "can be used to generate just one unique id" do
+      uids = WithGlobalUID.generate_many_uids(1)
+
+      assert_instance_of Array, uids
+      assert_equal uids.sort, uids
+      assert_equal uids.uniq, uids
```

Comment:
The last two assertions are not too reassuring, since they imply that the array may have multiple elements. I’d much rather replace those two lines with one: `assert_equal uids.size, 1`.